src/v3/package.json

```diff
@@ -61,6 +61,7 @@
     "@okta/odyssey-react-mui": "^0.17.0",
     "@okta/odyssey-react-theme": "^0.15.3",
     "@okta/okta-auth-js": "^7.2.0",
+    "@types/react-html-parser": "^2.0.2",
```

Comment:
Is dependencies the correct place for this? Or should the types only be added to devDependencies?

Reply:
should be devDeps since it's not used at runtime